devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -777,6 +781,16 @@ void HumanStateProvider::run()
                                         pImpl->baseVelocitySolution.getVal(4),
                                         pImpl->baseVelocitySolution.getVal(5)};
     }
+    {
+        iDynTree::KinDynComputations* computations = pImpl->kinDynComputations.get();
```

Comment:
please change the variable name from `computations` to something grounding like `kindyncomputations` to be more clear

Reply:
done!